native/game_backend/src/player.rs

```diff
@@ -92,8 +92,10 @@ impl Player {
     }
 
     pub fn add_action(&mut self, action: Action, duration_ms: u64) {
-        self.next_actions.push(action);
-        self.action_duration_ms += duration_ms;
+        if !self.actions.contains(&action) {
+            self.next_actions.push(action);
+            self.action_duration_ms += duration_ms;
+        }
```

Comment:
This looks like we really want some kind of set instead.

Reply:
Interesting enough, due to the small size of this list a Vec might be more efficient

Reply:
I'll work with @agustinesco to look into this in #64 as his reworking the actions in there